scripts/qiita

```diff
@@ -130,24 +130,31 @@ def load_study(owner, title, info):
               help="The parent artifacts from which the new artifact has been "
                    "processed",
               type=click.IntRange(1))
-@click.option('--command', required=False,
-              help="The command used to process the new artifact",
-              type=click.IntRange(1))
-@click.option('--parameter_set', required=False,
-              help="The parameter set used to process the new artifact",
+@click.option('--dflt_params', required=False,
```

Comment:
Maybe I just have not used this feature enough, but is that really more
useful? (not a rhetorical question).

On (Nov-25-15| 9:18), Antonio Gonzalez wrote:
> > @@ -130,24 +130,31 @@ def load_study(owner, title, info):
> >                help="The parent artifacts from which the new artifact has been "
> >                     "processed",
> >                type=click.IntRange(1))
> > -@click.option('--command', required=False,
> > -              help="The command used to process the new artifact",
> > -              type=click.IntRange(1))
> > -@click.option('--parameter_set', required=False,
> > -              help="The parameter set used to process the new artifact",
> > +@click.option('--dflt_params', required=False,
> 
> Yes, that's the reason of a new issue. Note that is possible to list all available commands/params with their unique identifier (int). 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1545/files#r45894093